packages/website/blog/2023-07-09-announcing-typescript-eslint-v6.md

```diff
@@ -63,7 +63,7 @@ As a result, we've reworked the configurations provided by typescript-eslint int
 
 - Functional rule configurations, for best best practices and code correctness:
   - **`recommended`**: Recommended rules that you can drop in without additional configuration.
-  - **`recommended-type-checked`**: Additional recommended rules that require type information.
+  - **`recommended-type-checked`**: Additional recommended rules that require type information.
```

Comment:
What's the deal with this diff?

Reply:
no idea, it showed up as different when I copy pasted the md file from the main branch. are they different kind of spaces or something? I tried copy pasting them into [this site](https://www.babelstone.co.uk/Unicode) and they both show up as normal spaces, so I'm not really sure

Reply:
`git checkout main -- packages/website/blog/2023-07-09-announcing-typescript-eslint-v6.md` fixed it for me. 🤷